cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -11,19 +11,35 @@
 from anndata.compat import DaskArray
 from anndata.experimental import read_dispatched, read_elem_as_dask
 from cellxgene_ontology_guide.ontology_parser import OntologyParser
-from dask.array import map_blocks
-from scipy import sparse
+from cellxgene_schema.matrix_utils import calculate_matrix_nonzero, check_non_csr_matrixes, determine_matrix_format
 from xxhash import xxh3_64_intdigest
 
 logger = logging.getLogger(__name__)
 
-SPARSE_MATRIX_TYPES = {"csr", "csc", "coo"}
-SUPPORTED_SPARSE_MATRIX_TYPES = {"csr"}
-
 KB = 1024
 MB = 1024 * KB
 GB = 1024 * MB
 
+"""
+Ideally, these methods should all only live within matrix_utils. However, we
+currently import these into single-cell-data-portal, so we need to keep these
+in here for backwards compatibility until we can refactor that.
+"""
+SPARSE_MATRIX_TYPES = {"csr", "csc", "coo"}
+SUPPORTED_SPARSE_MATRIX_TYPES = {"csr"}
+
+
+def get_matrix_format(matrix: DaskArray) -> str:
```

Comment:
i slightly renamed all the methods between `utils.py` and `matrix_utils.py` because i got a warning from python saying that i was redefining a variable on line 32 that was imported on line 14. it didn't actually seem like it caused an issue with test correctness, but it seemed a bit safer to just rename the methods entirely.